packages/babel-plugin/src/utils/evaluate-expression/index.tsx

```diff
@@ -40,158 +36,6 @@ const traverseIdentifier = (expression: t.Identifier, meta: Metadata) => {
   return createResultPair(value as t.Expression, updatedMeta);
 };
 
-/**
```

Comment:
All this removed code was handling member expression evaluation. I've tried to simplify it into `traverseMemberAccessPath`